src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,92 @@
-
-            <td className="is-vcentered">
-              <span className="icon" data-cy="iconCompleted">
-                <i className="fas fa-check" />
-              </span>
-            </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">illo est ratione doloremque quia maiores aut</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
+      {!todos.length ? (
+        <p className="notification is-warning">
+          There are no todos matching current filter criteria
+        </p>
+      ) : (
+        <table className="table is-narrow is-fullwidth">
+          <thead>
+            <tr>
+              <th>#</th>
+
+              <th>
                 <span className="icon">
-                  <i className="far fa-eye" />
+                  <i className="fas fa-check" />
                 </span>
-              </button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+              </th>
+
+              <th>Title</th>
+              <th> </th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {todos.map(todo => {
+              const { id, completed, title } = todo;
+
+              return (
+                <tr
+                  data-cy="todo"
+                  className={classNames(
+                    { 'has-background-info-light': id === currentTodo?.id },
+                  )}
+                  key={id}
+                >
+                  <td className="is-vcentered">{id}</td>
+                  <td className="is-vcentered">
+                    {completed && (
+                      <span className="icon" data-cy="iconCompleted">
+                        <i className="fas fa-check" />
+                      </span>
+                    )}
+                  </td>
+
+                  <td className="is-vcentered is-expanded">
+                    <p className={classNames(
+                      { 'has-text-danger': !completed },
+                      { 'has-text-success': completed },
+                    )}
+                    >
+                      {title}
+                    </p>
+                  </td>
+
+                  <td className="has-text-right is-vcentered">
+                    <button
+                      data-cy="selectButton"
+                      className="button"
+                      type="button"
+                      onClick={() => {
+                        dispatch(currentTodoActions.setTodo(todo));
+                      }}
+                    >
+                      <span className="icon">
+                        <i className={classNames({
+                          'far fa-eye-slash': currentTodo?.id === id,
+                        }, {
+                          'far fa-eye': currentTodo?.id !== id,
+                        })}
```

Comment:
```suggestion
                        <i className={classNames({
                          'far fa-eye-slash': isCurrentTodo,
                          'far fa-eye': !isCurrentTodo,
                        })}
```

Create constant for this expression